### PR TITLE
make sure home/user/upload dirs exist and have 755 perms

### PR DESCRIPTION
**Pull request by adiletmaratov**

### CODE REVIEW

File: CLOUDKITE.md [hunk 1]
```diff
@@ -24,9 +24,9 @@ ssh-keygen -t rsa -b 4096 -f /mnt/volume/config/ssh_host_rsa_key < /dev/null
 ```bash
 mkdir -p /mnt/volume/scripts
 cp set_permissions.sh /mnt/volume/scripts
+cp create_home_dirs.sh /mnt/volume/scripts
```
Comment (victortrac) [inline CLOUDKITE.md:27]: We should just move all of the custom scripts into a `startup-scripts` dir in this repo. Then you don't need custom instructions for each script.

File: docker-compose.yaml [hunk 2]
```diff
@@ -6,6 +6,7 @@ services:
       - /mnt/volume/config/users.conf:/etc/sftp/users.conf:ro
       - /mnt/volume/config/ssh_host_ed25519_key:/etc/ssh/ssh_host_ed25519_key
       - /mnt/volume/config/ssh_host_rsa_key:/etc/ssh/ssh_host_rsa_key
+      - /mnt/volume/scripts/create_home_dirs.sh:/etc/sftp.d/create_home_dirs.sh
```
Comment (victortrac) [inline docker-compose.yaml:9]: this should just be: 
```
- /mnt/volume/startup-scripts:/etc/sftp.d/
```

File: CLOUDKITE.md (hunk 1 above)
Comment (adiletmaratov) [reply CLOUDKITE.md:27]: Fixed it in https://github.com/cloudkite-io/sftp/pull/7

File: docker-compose.yaml (hunk 2 above)
Comment (adiletmaratov) [reply docker-compose.yaml:9]: Fixed it in https://github.com/cloudkite-io/sftp/pull/7

**Pull request merged**